test/e2e/35_skip_crds.bats

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bats
+
+function setup() {
+  # Load libraries in setup() to access BATS_* variables 
```

Comment:
Copying and pasting e2e to get going makes me feel dirty... I will have a look at what common stuff we can extract out at some point to make it easier to get started.